R/constructors.R

```diff
@@ -18,7 +18,8 @@
 #'  values should be inclusive or exclusive. If `values` is supplied,
 #'  and `inclusive` is `NULL`, `inclusive` will be set to `c(TRUE, TRUE)`.
 #'
-#' @param default A single value with the same class as `type` for the default
+#' @param default `r lifecycle::badge("deprecated")`
+#' A single value with the same class as `type` for the default
 #' parameter value. `unknown()` can also be used here.
```

Comment:
```suggestion
#' @param default `r lifecycle::badge("deprecated")`
#' No longer used. If a value is supplied, it will be ignored and
#' a warning will be thrown.
```